app/assets/javascripts/wizard/addon/components/wizard-step.js

```diff
@@ -26,9 +26,17 @@ export default Component.extend({
 
   @discourseComputed("step.displayIndex", "wizard.totalSteps")
   showNextButton(current, total) {
+    if (this.showConfigureMore === true) {
+      return false;
+    }
     return current < total;
   },
 
+  @discourseComputed("step.id")
+  backButtonLabel() {
+    return "wizard.back";
+  },
+
```

Comment:
If this is always `"wizard.back"` then there's no need for a computed property and this string can be moved to the template